docs/README_es.md

```diff
@@ -0,0 +1,220 @@
+
+<p align="center">
+    <img alt="GitHub last commit" src="https://img.shields.io/github/last-commit/CyFeng16/MVIMP" />
+    <img alt="GitHub issues" src="https://img.shields.io/github/issues/CyFeng16/MVIMP" />
+    <img alt="GitHub License" src="https://img.shields.io/github/license/cyfeng16/MVIMP" />
+    <img alt="Code style: black" src="https://img.shields.io/badge/code%20style-black-000000.svg" />
+</p>
+
+[English](/README.md) | [简体中文](README_zh-Hans.md) | Español
+
+# MVIMP
+
+**M**ixed **V**ideo and **I**mage **M**anipulation **P**rogram
+*(Programa de manipulación de imágenes y videos variados)*
+
+Entrenar un modelo de IA de buen rendimiento, es solo una parte de la historia, el hacerlo fácil de usar para otras personas, es otra. Por lo tanto, este repositorio intenta abarcar la capacidad de la IA para manipular multimedia, también, ¡Deseo que te diviertas!
+
+| Modelo |  Entrada | Salida |        Paralelo?        |
+|:--------:|:------:|:------:|:----------------------:|
+| [AnimeGAN](https://github.com/CyFeng16/MVIMP#animegan) | Imagenes | Imagenes |          Si          |
+|   [DAIN](https://github.com/CyFeng16/MVIMP#dain)   |  Video |  Video |          No         |
+|  [Photo3D](https://github.com/CyFeng16/MVIMP#photo3d) | Imágenes | Video | Si (No se recomienda) |
+| [DeOldify](https://github.com/CyFeng16/MVIMP#deoldify) | Imágenes | Imágenes |          Si          |
```

Comment:
Those internal links did not change.

Reply:
Oops! Forget that, I did not test it in the end, ok